config/nrfconnect/chip-module/Kconfig

```diff
@@ -18,6 +18,27 @@ rsource "../../zephyr/Kconfig"
 
 if CHIP
 
+config CHIP_WIFI_NRF
+	bool "Enable nrfconnect Wi-Fi support"
+	default n
+	select WIFI_NRF
+	imply NORDIC_SECURITY_BACKEND
```

Comment:
How about setting these three in `Kconfig.defaults` regardless of whether WiFi or Thread is used?

Reply:
These are selected by the OpenThread, hence I force them in case WiFi to align with the OT configuration. I mean, we can move these to the global scope but it would impact only WiFi builds anyway.

Reply:
Actually, that's minor. I just thought it would be cleaner if it was set unconditionally rather than making the impression that only WiFi needs it, but I assume we will need other rounds of the configuration cleanup after Wifi support gets more mature.